docs/background-color.md

```diff
@@ -52,15 +52,9 @@ For example, use `hover:s-bg-positive` to only apply the `s-bg-positive` utility
 </container>
 
 ```html
-<div
-  class="s-bg hover:s-bg-hover active:s-bg-active active:hover:s-bg-active-hover ..."
-></div>
-<div
-  class="s-bg-info hover:s-bg-info-hover active:s-bg-info-active active:hover:s-bg-info-active-hover ..."
-></div>
-<div
-  class="s-bg-warning hover:s-bg-warning-hover active:s-bg-warning-active active:hover:s-bg-warning-active-hover ..."
-></div>
+<div class="s-bg hover:s-bg-hover active:s-bg-active active:hover:s-bg-active-hover ..."></div>
+<div class="s-bg-info hover:s-bg-info-hover active:s-bg-info-active active:hover:s-bg-info-active-hover ..."></div>
```

Comment:
In the inspector it looks like `--w-s-color-background-info-active-hover` and `--w-s-color-background-warning-active-hover` are not found:
![background-color](https://github.com/warp-ds/css-docs/assets/41303231/2ff0c026-246c-4d26-92f3-dfcdd8fc8060)

Reply:
I haven't changed anything except formatting... but yes it seems to be bad examples here... will fix. 👍